### DISCUSSION TITLE
Warning before opening on older versions

### DISCUSSION THREAD

**Discussion by LoneDespair** [godotengine/godot-proposals#6268] [Engine Core]
I think it would nice,
if there's a warning popup when attempting to open a project on an older version

as there are things like the "unique name" which just disappears when being opened on an older version
I use different Godot versions for different projects, so this happens to me, a little too often

this would also be handy for open source projects, so they don't just silently fail,
and have to guess which version they are designed for

**Comment by Zireael07** [comment]
You can't open a 4.x project on 3.x anyway.

As for different 3.x versions, I have a feeling we had a discussion on that earlier but I can't find it in the list... @Calinou ?

**Reply by LoneDespair** [reply]
I only use 3.x

But this might still be useful between future 4.x versions

**Reply by Zireael07** [reply]
Paraphrasing from that other discussion I can't find but remember, it's not possible to know which 3.x version the project was created/opened on, so it's not possible to warn/prevent opening on older versions.

**Reply by Calinou** [reply]
`config_version` in `project.godot` does not distinguish between Godot 3.x versions after 3.2, so there is no way to do this. (`config_version` is only updated for breaking changes to the format.)

**Reply by LoneDespair** [reply]
I see, thank you for your time
